README.markdown

```diff
@@ -81,6 +81,30 @@ The Hypothesis LMS app is written for python 3 and uses Node.js and `yarn` for m
       need the generated <samp>Client ID</samp> and <samp>Client secret</samp>
       for setting your environment variables later.
 
+1. Also create a jwt_bearer auth client in h.
+
+   The LMS app also requires the `client_id` and `client_secret` from a
+   `jwt_bearer`-type auth client in h in order to generate authorization grant
+   tokens for logging in to h user accounts.
+
+   Go to <http://localhost:5000/admin/oauthclients/new> and create an auth
```

Comment:
Standard Markdown. It prevents things like trailing punctuation etc from being treated as part of the URL, and I prefer to use it even when not strictly necessary as it makes editing easier (if in the future the file is edited and some trailing punc gets in there). Also prefer it because it makes the Markdown more portable as original Markdown requires it - although it's common for Markdown renderers to linkify naked URLs I don't think that's actually part of original Markdown -- in the original Markdown implementation the angle brackets were required to linkify a URL.

You can do this with email addresses too (`<address@example.com>`) and Markdown will turn it into a mailto link and obfuscate the HTML address for you.

See https://daringfireball.net/projects/markdown/syntax#autolink

Reply:
Thanks for the tip. I'm surprised I hadn't learned about this until today.